docs/readthedocs/source/_toc.yml

```diff
@@ -52,6 +52,7 @@ subtrees:
                       - file: doc/LLM/Overview/KeyFeatures/inference_on_gpu
                       - file: doc/LLM/Overview/KeyFeatures/finetune
                       - file: doc/LLM/Overview/KeyFeatures/multi_gpus_selection
+                      - file: doc/LLM/Overview/Quickstart/benchmark_quickstart
```

Comment:
Also need to modify https://bigdl.readthedocs.io/en/latest/doc/LLM/Quickstart/index.html